test/conformance/adapters/cuda/kernel_tests.cpp

```diff
@@ -0,0 +1,270 @@
+// Copyright (C) 2022-2023 Intel Corporation
+// Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+// See LICENSE.TXT
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "kernel.hpp"
+#include <uur/fixtures.h>
+
+using cudaKernelTest = uur::urQueueTest;
+UUR_INSTANTIATE_DEVICE_TEST_SUITE_P(cudaKernelTest);
+
+inline constexpr size_t NumberOfImplicitArgsCUDA = 1;
```

Comment:
Nit: A comment that explains that this argument is used for the implicit global offset could be helpful